NewForce-at-Mountwest/bangazon-workforce-rat
# Create Employee - ticket #3

# Description

User will be able see a list of employees, and  create an employee by inputting their first and last name, select department from dropdown, and option to checkmark if employee is a supervisor on the create form.

Click Create green button to submit form / create new employee. 


## Related Ticket(s)
Ticket #3 - Create New Employee


## Steps to Test
Click on Employees
Employee List will appear
Click "Create New" link
Form will appear to enter the new employee's:
First Name
Last Name
Checkbox - Is Supervisor? - Checkmark if employee is a supervisor.
Dropdown list -  to select which department employee belongs to. 


# Checklist:
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] I have made corresponding changes to the documentation
